### PR TITLE
Added Crashlytics

### PR DESCRIPTION
**Pull request by f7deleon**
Added Crashlytics

### Related Trello cards:
https://trello.com/c/vHXh3wEs

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,5 +1,21 @@
+buildscript {
+    repositories {
+        maven { url 'https://maven.fabric.io/public' }
+    }
+
+    dependencies {
+        classpath 'io.fabric.tools:gradle:1.+'
```
Comment (FerrariBruno) [inline app/build.gradle:7]: maybe put a specific version instead of this

**Review by FerrariBruno — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (mirland) [reply app/build.gradle:7]: yep, you have to do it

File: app/build.gradle [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+    }
+}
+
 apply plugin: 'com.android.application'
```
Comment (mirland) [inline app/build.gradle:12]: rm

File: app/build.gradle [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
 
+apply plugin: 'io.fabric'
+
+repositories {
```
Comment (mirland) [inline app/build.gradle:15]: sort

File: app/build.gradle [hunk 4]
```diff
@@ -31,10 +47,15 @@ dependencies {
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
 
+    compile('com.crashlytics.sdk.android:crashlytics:2.9.4@aar') {
```
Comment (mirland) [inline app/build.gradle:50]: you shouldn't use compile

File: app/build.gradle [hunk 5]
```diff
@@ -31,10 +47,15 @@ dependencies {
...
+    }
+
     implementation "com.android.support:appcompat-v7:${ANDROID_SUPPORT_VERSION}"
     implementation 'com.android.support.constraint:constraint-layout:1.1.2'
     implementation "com.jakewharton.timber:timber:${TIMBER_SUPPORT_VERSION}"
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
 
     testImplementation 'junit:junit:4.12'
 }
+
```
Comment (mirland) [inline app/build.gradle:61]: rm

File: app/src/main/AndroidManifest.xml [hunk 6]
```diff
@@ -27,6 +27,13 @@
...
         <activity android:name=".CheatActivity" />
 
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="5969046a2f97c655545b350a8c8d3d3c611be9b5"
+            />
+
     </application>
 
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (mirland) [inline app/src/main/AndroidManifest.xml:37]: put it at the beginning of the class

File: app/src/main/java/com/example/felipe/geoquiz/CheatActivity.kt [hunk 7]
```diff
@@ -32,7 +36,7 @@ class CheatActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_cheat)
-
+        Fabric.with(this, Crashlytics())
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/CheatActivity.kt:39]: add method to initialize this and timber

File: app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt [hunk 8]
```diff
@@ -31,7 +32,7 @@ class QuizActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         Timber.d("onCreate(Bundle) called")
         setContentView(R.layout.activity_quiz)
-
+        Fabric.with(this, Crashlytics())
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt:35]: Shouldn't you do it only in the first activity?

Comment (mirland) [reply app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt:35]: I'm not sure, just a question

**Pull request closed without merging**